### ISSUE TITLE
Setting to upload results to S3 Bucket 

### ISSUE THREAD

**Issue by cooperq** [EFForg/rayhunter#78]
Need a setting to automatically upload results to an amazon s3 bucket (or similar). Should be able to specify bucket location in config file.

**Comment by cooperq** [comment]
This is pretty straightforward technically, put a URL in the config file and have qmdl files automatically upload to that url. The questions are around communicating the potential risks of this to rayhunter users. We will want to review this with a UI expert probably.

**Comment by blanchardjeremy** [comment]
Do you mean a single centralized S3 bucket for everything? Or user-configured buckets?

If it's user-configured - Would it make sense to have them end-to-end encrypted by default for security?

**Comment by cooperq** [comment]
Definitely not a publicly readable S3 bucket, and to clarify this is not something I want to take a look at pull requests on yet. This issue needs a lot more thought about how to do it right and preserve privacy.

**Comment by untitaker** [comment]
do you want to track this separately from #108 ?

**Comment by cooperq** [comment]
I think so? for this setting I imagine that its an endpoint people would host themselves, this is a quality of life thing for someone who may have several devices in the field, whereas #108 is more about getting data to EFF.

**Comment by cooperq** [comment]
https://github.com/9001/copyparty/ this is probably what we want to support as a file server!

**Comment by simonft** [comment]
I'm guessing this depends on #549 since the data being transmitted is pretty sensitive. Unless we want to add the option to encrypt the data before upload, but that might increase the binary size too much.

**Comment by cooperq** [comment]
I don't think it necessarily has to but I probably wouldn't want to release this one until #549 was finished I suppose.
